### ISSUE TITLE
Work with keypoints for recognize some poses

### ISSUE THREAD

**Issue by abduladolph23** [Deci-AI/super-gradients#2008]
### 💡 Your Question

Hi, i want to recognize when a person is raising their hand. My idea is to calculate when the keypoint of the hand is greater than the keypoint of shoulder (something like that). But I'm not able to get the keypoints of the body. Please i really need help :( .

I'm using supergradients -> 3.7.1

And the way i'm doing the recognition is:
`
import super_gradients
import torch

from super_gradients.training import models
from super_gradients.common.object_names import Models

device = 'cuda' if torch.cuda.is_available() else 'cpu'

model = models.get(Models.YOLO_NAS_POSE_N, pretrained_weights="coco_pose")
model = model.to(device)

model.predict_webcam(conf=.75)

`

**Comment by guor8bu** [comment]
Hi, Did you find the solution? I am trying to get them as well, at least the coordinate of all key points, don't need a specific one
